src/main/java/pl/intern/touk/internTouK/controller/ScreeningController.java

```diff
@@ -0,0 +1,90 @@
+package pl.intern.touk.internTouK.controller;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import pl.intern.touk.internTouK.dto.ScreeningDTO;
+import pl.intern.touk.internTouK.model.Seat;
+import pl.intern.touk.internTouK.model.SeatState;
+import pl.intern.touk.internTouK.model.Ticket;
+import pl.intern.touk.internTouK.model.TicketType;
+import pl.intern.touk.internTouK.repository.ScreeningRepository;
+import pl.intern.touk.internTouK.converter.ScreeningConverter;
+
+
+import javax.validation.Valid;
+import java.time.LocalDateTime;
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/screening")
+public class ScreeningController {
+    @Autowired
+    ScreeningRepository screeningRepository;
+
+    @GetMapping("/{id}")
+    ScreeningDTO getScreening(@PathVariable Long id) {
+        return ScreeningConverter.toScreeningDTO(screeningRepository.findById(id).get());
+    }
+
+    @GetMapping
+    Collection<ScreeningDTO> getScreenings() {
+        return ScreeningConverter.toScreeningDTO(screeningRepository.findAll());
+    }
+
+    @GetMapping("/{year}/{month}/{day}/{hour}/{minute}")
+    Collection<ScreeningDTO> getScreenings(@PathVariable Integer year, @PathVariable Integer month, @PathVariable Integer day, @PathVariable Integer hour, @PathVariable Integer minute) {
+        return ScreeningConverter.toScreeningDTO(screeningRepository.findByDateGreaterThan(LocalDateTime.of(year, month, day, hour, minute + 15)));
+    }
+
+    @GetMapping("/put")
```

Comment:
Rozumiem, że to jakiś testowy endpoint? Dobrze byłoby czegoś takiego nie zostawiać